### PR TITLE
SAR-9991 Create new OTI page

### PR DESCRIPTION
**Pull request by woollett-r**

### CODE REVIEW

File: it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/overseasControllers/CaptureOverseasTaxIdentifierControllerISpec.scala [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+      }
+    }
+
+  }
+
+  "POST /overseas-tax-identifier" when {
+
+    "the user selects 'Yes' and the tax identifier is correctly formatted" should {
+
+      "redirect to Check Your Answers" in {
```
Comment (refaque-ahmad) [inline it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/overseasControllers/CaptureOverseasTaxIdentifierControllerISpec.scala:80]: Believe it goes to OTI country page which will be done in separate ticket. Would be nice put a comment something

File: app/uk/gov/hmrc/minorentityidentificationfrontend/forms/CaptureOverseasTaxIdentifierForm.scala [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+  private val overSeasTaxIdentifierRadioKey: String = "tax-identifier-radio"
+  private val overSeasTaxIdentifierKey: String = "tax-identifier"
+
+  private val noSelectionMadeErrorMsg: String = "error.no_tax_identifier_selection"
+  private val overSeasTaxIdentifierNotEnteredErrorMsg: String = "error.no_tax_identifier"
+  private val overSeasTaxIdentifierTooLongErrorMsg: String = "error.invalid_tax_identifier_length"
+  private val overSeasTaxIdentifierInvalidCharsErrorMsg: String = "error.invalid_tax_identifier"
+
+  private val overSeasTaxIdentifierFormatter: Formatter[Option[String]] = new Formatter[Option[String]]  {
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply app/uk/gov/hmrc/minorentityidentificationfrontend/forms/CaptureOverseasTaxIdentifierForm.scala:38]: Done

File: app/uk/gov/hmrc/minorentityidentificationfrontend/views/overseasCompanyViews/capture_overseas_tax_identifier_page.scala.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                content = Text(messages("app.common.yes")),
+                value = Some(YesNo.Yes),
+                conditionalHtml = Some(govukInput(
+                                       Input(
+                                               id = "tax-identifier",
+                                               name = "tax-identifier",
+                                               label =  Label(
+                                                               content = HtmlContent(messages("tax-identifiers.field_1")),
+                                                               classes = "govuk-label govuk-!-font-weight-bold"
+                                                              ),
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply app/uk/gov/hmrc/minorentityidentificationfrontend/views/overseasCompanyViews/capture_overseas_tax_identifier_page.scala.html:67]: That is a good spot!

[the remark this answers is not in the document]
Comment (woollett-r) [reply app/uk/gov/hmrc/minorentityidentificationfrontend/views/overseasCompanyViews/capture_overseas_tax_identifier_page.scala.html:67]: Done

File: it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/overseasControllers/CaptureOverseasTaxIdentifierControllerISpec.scala (hunk 1 above)
Comment (woollett-r) [reply it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/overseasControllers/CaptureOverseasTaxIdentifierControllerISpec.scala:80]: Done

**Review by refaque-ahmad — APPROVED** [review]

**Pull request merged**